doc/release/upcoming_changes/24978.deprecation.rst

```diff
@@ -0,0 +1,5 @@
+Use of keyword arguments ``x`` and ``y`` with functions
+`numpy.testing.assert_array_equal` and
+`numpy.testing.assert_array_almost_equal`
+has been deprecated. Pass the first two arguments as positional arguments,
+instead.
```

Comment:
This probably doesn't render well, either make it a bullet point with an initial `*` and indentation, or give it a title with `----`.